app/index.js

```diff
@@ -202,7 +206,7 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   generateYoRc: function() {
-    this.log('Generating .yo-rc.json');
+    g.log('Generating .yo-rc.json');
```

Comment:
In this case, two things to consider:
1. What is 'this`?  'this` in JavaScript is a reserved word and has a special meaning.  `this.log` may be `console.log` may be client specific function.  `g.log` is a `console.log` equivalent.
2. `.yo-rc.json` should not be translated.  The translator will probably figure out, but we have a notation to specify `don't translate."  We should use it here.

Probably the best globalization is:

`this.log(g.f('Generating {{.yo-rc.json}}'));`